packages/react/counter/src/Counter.tsx

```diff
@@ -118,18 +136,19 @@ export function Counter(props: CounterProps) {
     async function setup() {
       if (isStatic && value) {
         setDataValue(value)
-        return
       }
 
-      const fetchedValue = await fetchData()
+      if (!isStatic) {
+        const fetchedValue = await fetchData()
 
-      if (!fetchedValue) {
-        setHasError(true)
-        console.error(`QueryError: Your metric ${query?.metric} returned undefined or a \`null\` value.`)
-        return
-      }
+        if (typeof fetchedValue === 'undefined') {
```

Comment:
This is related to an issue we found in the sample app.
The Counter can return `0` and that would be a normal case, but since `0` is a falsy value it would incorrectly throw this error